rfcXXX-github-issues-workflow.md

```diff
@@ -0,0 +1,92 @@
+
+## Terms
+
+In the context of this document below terms are used:
+
+* **Chef Users:** End users of Chef who participate in this workflow by filing **issues**.
+* **Contributors:** Users of Chef who would like to improve Chef. They participate in this workflow by submitting **contributions** to Chef.
+* **Chef Maintainers:** Developers of Chef who are responsible for one or more **subcomponents**.
+* **Lieutenants:** Developers of Chef who lead the maintenance of a subcomponent.
+* **Issue**: A filed **Github Issue** which can be a bug report, feature request, style change request.
+* **Contribution**: A filed **Github Pull Request** (PR).
+* **Subcomponents**: A sub-area of Chef project which is lead by a **Lieutenant** and a set of **maintainers**.
+
+## Goals
+
+Chef Users can:
+
+* Be aware of the cause & the next steps for the issues they have filed.
+
+Contributors can:
+
+* Identify the state of their contribution.
+
+Chef Maintainers can:
+
+* See the list of contributions they need to review in a single page.
+* See the list of issues they need to investigate in a single page.
+
+
+## Workflow
+
+![Chef GitHub Issues Workflow](./GithubIssuesWorkflow.png)
+
+
+## Notes
+
+### Common Steps for Issues & PRs
+
+The first step of Chef Github Issues Workflow is to identify the subcomponent for which the issue or PR belongs to. Once the subcomponent is identified, the rest of the workflow is the responsibility of the maintainers of the identified subcomponent.
+
+### Issues Workflow
+
+The issues workflow can be broken down into these high level steps:
+
+1. Understand the issue and determine the resolution.
+1. Identify the major version the issue should be fixed in.
+1. Label the issue to indicate completeness of **Triage** process.
+
+Any code or contributions associated with the issue will follow the
+contribution workflow documented below.
+
+#### Rules
+
+* If an issue is not commented on for more than a year, the issue is retired by closing.
+
+### Contribution Workflow
+
+The contribution workflow can be broken down into these high level steps:
+
+1. Maintainers check the contribution and give **:+1:** if the code is looking good, or leave a comment that clearly identifies what is needed for the contribution to move forward.
+1. Once at least two maintainers **:+1:** a PR, the last reviewer should:
+  * If clean up, such as a Changelog entry or a rebase, is required, mark the PR with the `Ready to Merge` label.
+  * If no further work is required, merge the PR.
+1. PRs marked with `Ready to Merge` will be merged by maintainers of the subcomponent after including the required documentation updates.
+1. `Waiting for Contributor Response` label indicates that an action is needed from the contributor.
+1. `Needs Maintainer Review` label indicates that an action is needed from Chef maintainers.
+
+#### Rules
+
+* PRs labeled with `Waiting for Contributor Response` and not updated for more than a month are closed.
```

Comment:
The "more than a month" period here is too short.  People are busy and many times PRs sit around but eventually do get worked on and merged.  Closing them is not going to encourage people to complete them.

There is also now the "Incomplete Contribution" tag to apply to PRs which have been closed (often due to needing specs).  Some of those should have the Easy tag added to them.  If the community could rescue some of these it would be useful work.